db/migrate/20160114011126_create_indices.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIndices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :indices do |t|
+      t.datetime :finalized_at
+      t.string :awrvi_version
```

Comment:
yeah, that would be the idea now that the root node holds the version of the awrvi info